src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -107,6 +106,42 @@ Self-service VPC peering setup is not supported for CockroachDB {{ site.data.pro
 
 To establish an AWS PrivateLink connection, refer to [Managing AWS PrivateLink for a cluster]({% link cockroachcloud/aws-privatelink.md %}). After the connection is established, you can use it to [connect to your cluster](#connect-to-your-cluster).
 
+#### Azure Private Link
+
+{{site.data.alerts.callout_success}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+1. Navigate to your cluster's **Networking > Private endpoint** tab.
+1. Click **Add a private endpoint**. Copy the value provided for **Alias**. Do not close this browser window.
+1. In a new browser window, log in to Azure Console and create a new private endpoint for your cluster.
+    - Set the connection method to “by resource ID or alias”.
+    - Set the resource ID to the **Alias** you previously copied. For details, refer to [Create a private endpoint](https://learn.microsoft.com//azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip) in the Azure documentation.
+
+    After the private endpoint is created, open it, click **Properties**, and copy its Resource ID.
+
+    {{site.data.alerts.callout_info}}
+    Copy the resource ID for the private endpoint you just created, not for the Private Link resource itself.
+    {{site.data.alerts.end}}
+
+    Do not close this browser window.
+1. Return to the CockroachDB {{ site.data.products.cloud }} Console browser tab and click **Next**.
+1. Paste the resource ID for the Azure private endpoint, then click **Validate**. If validation fails, verify the resource ID and try again. If you encounter the error `This resource is invalid`, be sure that you are using the resource ID for the Azure private endpoint, rather than the resource ID for Azure Private Link itself.
```

Comment:
> rather than the resource ID for Azure Private Link itself

I think there might be any number of things that the customer puts into the box. Can we direct the customer to look at the properties page of their endpoint? There's only one thing on that page that looks like it fits, and it's the correct thing.

<img width="1168" alt="image" src="https://github.com/cockroachdb/docs/assets/4492794/e025d468-dc94-4272-bbbc-cb2de0c3810e">